catalog/transformer/catalog-transformer-tika-input/src/main/resources/OSGI-INF/blueprint/blueprint.xml

```diff
@@ -14,13 +14,21 @@
  -->
 <blueprint xmlns="http://www.osgi.org/xmlns/blueprint/v1.0.0">
     <!-- The tika input transformer programmatically registers itself as a service -->
-    <bean id="tikaTransformer" class="ddf.catalog.transformer.input.tika.TikaInputTransformer">
+    <bean id="tikaTransformer" class="ddf.catalog.transformer.input.tika.TikaInputTransformer" init-method="populateMimeTypeMap">
         <argument ref="blueprintBundleContext"/>
         <argument ref="commonMetacardType"/>
+        <property name="someMetacardType" ref="someMetacardType"/>
+        <property name="jpegMetacardType" ref="jpegMetacardType"/>
+        <property name="excelMetacardType" ref="excelMetacardType"/>
+        <property name="mp4MetacardType" ref="mp4MetacardType"/>
+        <property name="mpegMetacardType" ref="mpegMetacardType"/>
+        <property name="officeDocMetacardType" ref="officeDocMetacardType"/>
+        <property name="pdfMetacardType" ref="pdfMetacardType"/>
+        <property name="powerpointMetacardType" ref="powerpointMetacardType"/>
     </bean>
 
     <reference id="commonMetacardType" interface="ddf.catalog.data.MetacardType"
-               filter="(name=common)"
+               filter="(name=fallback.common)"
```

Comment:
After speaking with @pklinef , it was determined that there would be value in having the metacard names include the type of file ingested along with which parser was being used so that a user could differentiate two metacards created from the same product, but from different transformers. In that scenario, a user could have two metacards for the same product but with different attributes and no way for the user to determine why the attributes were different. 
“Fallback" was selected for the tikaInputTransformer since it is the "fallback option” to ingest a product in ddf.